gitindex/index.go

```diff
@@ -403,17 +403,18 @@ func IndexGitRepo(opts Options) error {
 
 		tree, err := commit.Tree()
 		if err != nil {
+			return fmt.Errorf("commit.Tree: %w", err)
 			return err
 		}
 
 		ig, err := newIgnoreMatcher(tree)
 		if err != nil {
-			return err
+			return fmt.Errorf("newIgnoreMatcher: %w", err)
 		}
 
 		files, subVersions, err := TreeToFiles(repo, tree, opts.BuildOptions.RepositoryDescription.URL, repoCache)
 		if err != nil {
-			return err
+			return fmt.Errorf("TreeToFiles: %w", err)
```

Comment:
nit: idiomatic go has error messages not start with a capital

Reply:
I think this is a good exception, because it refers to a function name. The standard library has a lot of those examples: https://sourcegraph.com/search?q=context:global+r:github.com/golang/go%24+fmt.Errorf%5C%28%22%5BA-Z%5D+-file:_test%5C.go%24+&patternType=regexp&case=yes